modules/ecs/cluster/main.tf

```diff
@@ -7,12 +7,9 @@ data "aws_caller_identity" "current" {}
 resource "aws_ecs_cluster" "ecs_cluster" {
   name = "nerves-hub-${var.environment}"
 
-  dynamic "setting" {
-    for_each = var.settings == "containerInsights" ? [var.settings] : []
```

Comment:
Ah, it may be due to the dynamic block only creating or not creating this block rather than being able to disable the setting after it's created. 

We could also do something like this. But it'll mean more parameters to manage and still would need to be updated if there were another setting in the future that was added. 🤷‍♀️ 
```
  dynamic "setting" {
    for_each = var.settings == "containerInsights" ? [var.settings] : []
    content {
      name  = setting.value
      value = var.container_insights ? "enabled" : "disabled"
    }
  }
```

![Screen Shot 2021-06-10 at 4 39 08 PM](https://user-images.githubusercontent.com/69476188/121610245-6d780d00-ca0a-11eb-8b13-9b67930a6808.png)